ui/src/pages/program/program.less

```diff
@@ -301,8 +301,11 @@ display      : block;
 width        : 100%;
 text-overflow: ellipsis;
 overflow     : hidden;
-white-space  : nowrap;
 min-width    : 85px;
+display: -webkit-box;
+white-space: normal;
+-webkit-box-orient: vertical;
+-webkit-line-clamp: 2;
 }
 
 .program .program_osobniTyp::before {
```

Comment:
Tohle je koukám nějaká Romanovina. Potřeboval styl pro admin a soubor zduplikoval. Poptám se ho na to.
Změnu jsi propsal do obou souborů správně, nějakou dobu asi budeme muset přežít s dvojím kódem.